cpp/data/installer/ub_tools.sql

```diff
@@ -39,8 +39,9 @@ CREATE TABLE zeder_journals (
 
 -- Table to be used w/ our validate_harvested_records tool:
 CREATE TABLE metadata_presence_tracer (
-    zeder_journal_id INT(11) UNSIGNED,
+    journal_id INT(11) UNSIGNED,
     marc_field_tag CHAR(3) NOT NULL,
+    marc_subfield_code CHAR(1) NOT NULL,
```

Comment:
any DEFAULT? (what happens to old values? if we throw all of them away, there needs to be a DELETE statement in the sql_update before adding the column)

Reply:
DONE

Reply:
OK, this should also be discussed with @socheres when merging